src/PaginationLink.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+const propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string,
+  next: PropTypes.bool,
+  previous: PropTypes.bool,
+  tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+};
+
+const defaultProps = {
+  tag: 'a',
+};
+
+const PaginationLink = (props) => {
+  const {
+    children,
+    className,
+    next,
+    previous,
+    tag: Tag,
+    ...attributes,
+  } = props;
+
+  const classes = classNames(
+    className,
+    'page-link'
+  );
+
+  return (
+    <Tag {...attributes} className={classes}>
```

Comment:
Can you add the aria label, the aria hidden attr and sr-only label for accessibility?

```
<a class="page-link" href="#" aria-label="Next">
    <span aria-hidden="true">&raquo;</span>
    <span class="sr-only">Next</span>
</a>
```